synapseclient/models/mixins/table_operator.py

```diff
@@ -1891,181 +2240,140 @@ async def main():
             values = DataFrame(values)
         elif isinstance(values, str):
             values = csv_to_pandas_df(filepath=values, **kwargs)
-        elif isinstance(values, DataFrame) or isinstance(values, str):
-            values = values.copy()
+        elif isinstance(values, DataFrame):
+            pass
```

Comment:
is this missing some functionality?

Reply:
no. This is just to make sure it doesn't fall into the

```
        else:
            raise ValueError(
                "Don't know how to make tables from values of type %s." % type(values)
            )
```

block. If they're passing in a `DataFrame` we don't need to do anything to prep the `values` argument for usage in the method.